File: C/Simple-Programming-Problems/001_Elementary/004_Num sum/Numsum.c
#include <stdio.h>

/*
* Task:
* Write a program that asks the user for a number n and prints the sum of the numbers 1 to n
*/

int main(int argc, char * argv[])
{
    int numberBuffer=0;
    int isDebug=0;
    
    printf("Enter an integer: ");
    scanf("%d",&numberBuffer);

    int sum=0;
    for(int i=numberBuffer;i>0;i--)
    {
        if(isDebug)
        {
            printf("[DEBUG]: Sum is %d + %d\n",sum,i);
        }
        sum+=i;
    }
    printf("The sum of all numbers in %d is: %d \n",numberBuffer,sum);

    return 0;
}

File: Python/Simple-Programming-Problems/001_Elementary/004_Num sum/NumSum.py
#!/bin/python3

"""
Task:
Write a program that asks the user for a number n and prints the sum of the numbers 1 to n
"""

numberBuffer=0
isDebug=False

numberBuffer=int(input("Enter an integer: "))

sum=0
for i in range(numberBuffer):
    if(isDebug):
        print(f"[DEBUG]: Sum is {sum} + {i}")
    sum+=i

print(f"The sum of all numbers in {numberBuffer} is: {sum}")